htdocs/comm/propal/card.php

```diff
@@ -114,7 +114,7 @@
 $usercancreate = $user->rights->propal->creer;
 $usercandelete = $user->rights->propal->supprimer;
 
-$usercanclose = ((empty($conf->global->MAIN_USE_ADVANCED_PERMS) && $usercancreate) || (!empty($conf->global->MAIN_USE_ADVANCED_PERMS) && !empty($user->rights->propal->propal_advance->close)));
+$usercanclose = $user->rights->propal->cloturer;
```

Comment:
This permission does not exists. Try to disable module and reenable to set correctly your permission in your database.

Reply:
![Screen Shot 2021-06-04 at 2 03 33 PM](https://user-images.githubusercontent.com/29521447/120759971-c690fe00-c53d-11eb-85d7-6a6f94f62d3e.png)

**modPropale.class.php line 156-159**
$this->rights[$r][0] = 26; // id de la permission
  156         $this->rights[$r][1] = 'Close commercial proposals'; // libelle de la permission
  157         $this->rights[$r][2] = 'd'; // type de la permission (deprecie a ce jour)
  158         $this->rights[$r][3] = 0; // La permission est-elle une permission par defaut
  159         $this->rights[$r][4] = 'cloturer';

IT is no have $user->rights->propal->propal_advance->close
just $user->rights->propal->cloturer

![Screen Shot 2021-06-04 at 2 03 33 PM](https://user-images.githubusercontent.com/29521447/120760709-a0b82900-c53e-11eb-9028-a24521bbb65f.png)

Reply:
What is your Dolibarr version ?

Reply:
13.02 and was update from 8.0

Reply:
You pushed your PR has a fix of develop branch. But into develop branch the permission "cloturer" does not exists anymore, probably because a fix for the same bug has already been pushed int version develop, fixing this differently.